src/Network-Kernel/NetNameResolver.class.st

```diff
@@ -160,7 +160,7 @@ NetNameResolver class >> localHostAddress [
 	| primAddress |
 
 	self initializeNetwork.
-	primAddress := self primLocalAddress.
+	[ primAddress := self primLocalAddress ] on: Error do: [ :err | primAddress := #[0 0 0 0] ].
```

Comment:
we should replace general error with PrimitiveFailed

Reply:
Understand. Should I make another pull request? Or - how to fix a pull request? :)

Reply:
@petr-fischer I have modified the PR directly, so we can integrate the change sooner. 
Sorry of the intromission, with @pavel-krivanek we considered that the change is good and we wanted to merge it.

To modify a PR you just commit new commits in the branch you were working and push the changes. 

Again this PR should be ready to merge, but for the next you can do like that. 

Thanks for contributing!